src/components/progress-bar/__test__/index.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from './../index';
+
+describe( 'The ProgressBar component', () => {
+	it( 'renders without crashing', () => {
+		const div = document.createElement( 'div' );
+		ReactDOM.render( <ProgressBar></ProgressBar>, div );
+	} );
+
+	it( 'shows the classnames of the outer div correctly', () => {
+		const { getByTestId } = render( <ProgressBar></ProgressBar> );
+		expect( getByTestId( 'progress-bar-outer' ) ).toHaveClass(
+			'wc-free-shipping-progress-bar__outer'
+		);
+	} );
+
+	it( 'shows the classnames of the inner div correctly', () => {
+		const { getByTestId } = render( <ProgressBar></ProgressBar> );
+		expect( getByTestId( 'progress-bar-inner' ) ).toHaveClass(
+			'wc-free-shipping-progress-bar__inner'
+		);
+	} );
+
+	it( 'shows the style for a full bar correctly', () => {
+		const { getByTestId } = render(
+			<ProgressBar freeShippingFrom="4" currentTotal="4"></ProgressBar>
+		);
+		expect( getByTestId( 'progress-bar-inner' ) ).toHaveStyle(
```

Comment:
From React testing library:
> it is recommended to use [test id] only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible.

Which is why I would recommend trying with something stable but isn't a test id.

Reply:
Given that I'm able to target all elements via CSS classes, I removed the `data-testid` attributes completely.